compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -1013,6 +1014,25 @@ class ResolverImpl(
             .plus(javaPackageToClassMap.getOrDefault(packageName, emptyList()).asSequence())
     }
 
+    @KspExperimental
+    override fun getSubpackagesOf(packageName: String): Sequence<String> {
+        fun PackageViewDescriptor.subPackages(): Sequence<PackageViewDescriptor> = memberScope
+            .getContributedDescriptors(DescriptorKindFilter.PACKAGES)
+            .asSequence()
+            .filterIsInstance<PackageViewDescriptor>()
+            .filterNot { it == this@subPackages }
```

Comment:
May I know why `.filterIsInstance<PackageViewDescriptor>()` and `.filterNot { it == this@subPackages }` are needed?